tests/schemas/issues.esdl

```diff
@@ -68,7 +68,7 @@ scalar type issue_num_t extending std::str;
 
 type Comment extending Text, Owned {
     required link issue -> Issue;
-    link parent -> Comment;
+    optional link parent -> Comment;
```

Comment:
I sprinkled a few `optional` declarations in this file as a form of testing the SDL support.